osm-seed/values.yaml

```diff
@@ -132,6 +132,7 @@ web:
     OVERPASS_URL: 'overpassApi'
     OAUTH_CLIENT_ID: 'abcd' # Create client ID and key as OAuth2 Application
     OAUTH_KEY: 'xyz'
+    ID_APPLICATION: 'xyz'
```

Comment:
Is this needed here?

Reply:
yes, to pass as an example and  to show what env var are required.